activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -61,7 +61,7 @@ def test_schema_dump
 
   def test_schema_dump_uses_force_cascade_on_create_table
     output = dump_table_schema "authors"
-    assert_match %r{create_table "authors", force: :cascade}, output
+    assert_match %r{create_table "authors", .*?force: :cascade}, output
```

Comment:
Well, the option here would be to regenerate schema.rb from existing database created by rails 4 , but it won't be very obvious for the user, I guess.